cmd/fakerp/main.go

```diff
@@ -0,0 +1,21 @@
+package main
+
+import (
+	"os"
+
+	"github.com/sirupsen/logrus"
+
+	"github.com/openshift/openshift-azure/pkg/fakerp"
+)
+
+func main() {
+	logrus.SetLevel(logrus.DebugLevel)
+	logrus.SetFormatter(&logrus.TextFormatter{FullTimestamp: true})
+	log := logrus.NewEntry(logrus.StandardLogger())
+
+	// TODO: Validate all required env variables are in place
+
+	log.Info("starting the fake resource provider")
+	s := fakerp.NewServer(log, os.Getenv("RESOURCEGROUP"), "localhost:8080")
```

Comment:
Does this have to be hardcoded on port 8080? Can't the port be randomized in one of the upper ephemeral ports for the duration of the run?

Reply:
Why do we care about randomizing it?